tower-http/src/timeout/body.rs

```diff
@@ -0,0 +1,379 @@
+//! Middleware that applies a timeout to bodies.
+//!
+//! Bodies must produce data at most within the specified timeout.
+//! If they are inactive, an error will be generated.
```

Comment:
Not sure what this means, how about 
```suggestion
//! If the body does not produce a requested data frame within the timeout period it will return an error.
```